### ISSUE TITLE
Feature Request: Pandoc in-text citation

### ISSUE THREAD

**Issue by ZBiener** [chrisgrieser/alfred-bibtex-citation-picker#18]
### Feature Requested

First, thanks for the great workflow. Second, those who use author-date citation style often use Pandoc's in-text citation format. This includes the citekey, but drops the square brackets surrounding it. So, where a "standard" citation has the form [@blah, p. 12], an in-text citation is just @blah [p. 12]. It would be nice to have this option.

I've modified the alfred workflow to do this for myself, but I just thought I'd mention the idea. If you are interested in my own hack, I'd be glad to share it. But I imagine you have a more elgant way of achieving this.

### Checklist

- [X] The feature would be useful to more users than just me.

**Comment by ZBiener** [comment]
One clarification: the idea is not to switch the citation format entirely (as one would switch from pandoc to latex citations). Rather, it is to have an on-the-fly choice between in-text and paranthetical citations.

**Comment by chrisgrieser** [comment]
Good suggestion. Added in the latest release, triggered via `⌘⇧ + ↵`. For the specifics, [see the updated docs on citation actions](https://github.com/chrisgrieser/alfred-bibtex-citation-picker#citation-actions).

(You will to update to the latest release by manually downloading it at GitHub, since I had to remove the auto-update mechanism for the upcoming Alfred Gallery. Auto-updating will be available again via Alfred in 1-2 days when the workflow is published in the Gallery.)
